consensus/src/counters.rs

```diff
@@ -78,166 +58,167 @@ pub static ref EPOCH_CHANGE_DEQUEUED_MSGS: IntCounter = OP_COUNTERS.counter("epo
 /// (calculated as the overall execution time of a block divided by the number of transactions).
-pub static ref TXN_EXECUTION_DURATION_S: DurationHistogram = OP_COUNTERS.duration_histogram("txn_execution_duration_s");
+pub static ref TXN_EXECUTION_DURATION_S: DurationHistogram = DurationHistogram::new(register_histogram!("libra_consensus_txn_execution_duration_s", "Histogram of per-transaction execution time of non-empty blocks (calculated as the overall execution time of a block divided by the number of transactions).").unwrap());
 
-/// Histogram of execution time (ms) of empty blocks.
-pub static ref EMPTY_BLOCK_EXECUTION_DURATION_S: DurationHistogram = OP_COUNTERS.duration_histogram("empty_block_execution_duration_s");
+/// Histogram of execution time of empty blocks.
+pub static ref EMPTY_BLOCK_EXECUTION_DURATION_S: DurationHistogram = DurationHistogram::new(register_histogram!("libra_consensus_empty_block_execution_duration_s", "Histogram of execution time of empty blocks.").unwrap());
 
 /// Histogram of the time it takes for a block to get committed.
 /// Measured as the commit time minus block's timestamp.
-pub static ref CREATION_TO_COMMIT_S: DurationHistogram = OP_COUNTERS.duration_histogram("creation_to_commit_s");
+pub static ref CREATION_TO_COMMIT_S: DurationHistogram =DurationHistogram::new(register_histogram!("libra_consensus_creation_to_commit_s", "Histogram of the time it takes for a block to get committed. Measured as the commit time minus block's timestamp.").unwrap());
 
 /// Duration between block generation time until the moment it gathers full QC
-pub static ref CREATION_TO_QC_S: DurationHistogram = OP_COUNTERS.duration_histogram("creation_to_qc_s");
+pub static ref CREATION_TO_QC_S: DurationHistogram = DurationHistogram::new(register_histogram!("libra_consensus_creation_to_qc_s", "Duration between block generation time until the moment it gathers full QC").unwrap());
 
 /// Duration between block generation time until the moment it is received and ready for execution.
-pub static ref CREATION_TO_RECEIVAL_S: DurationHistogram = OP_COUNTERS.duration_histogram("creation_to_receival_s");
+pub static ref CREATION_TO_RECEIVAL_S: DurationHistogram = DurationHistogram::new(register_histogram!("libra_consensus_creation_to_receival_s", "Duration between block generation time until the moment it is received and ready for execution.").unwrap());
 
 ////////////////////////////////////
 // PROPSOSAL/VOTE TIMESTAMP COUNTERS
 ////////////////////////////////////
 /// Count of the proposals that passed the timestamp rules and did not have to wait
-pub static ref PROPOSAL_NO_WAIT_REQUIRED_COUNT: IntCounter = OP_COUNTERS.counter("proposal_no_wait_required_count");
+pub static ref PROPOSAL_NO_WAIT_REQUIRED_COUNT: IntCounter = register_int_counter!("libra_consensus_proposal_no_wait_required_count", "Count of the proposals that passed the timestamp rules and did not have to wait").unwrap();
```

Comment:
would it make send to make this count name as `libra_consensus_proposal_count` and have state = no_wait_required / wait_was_required / wait_exceeded ?

Reply:
Done